app/config/AppConfig.scala

```diff
@@ -30,7 +30,7 @@ class AppConfig @Inject()(
   val runModeConfiguration: Configuration,
   val environment: Environment,
   servicesConfig: ServicesConfig,
-  @Named("appName") appName: String
+  @Named("appName") val appName: String
```

Comment:
in case if you will be able to inject just appName you don't need val here

Reply:
This has been addressed